### ISSUE TITLE
'assemble-windows' CI job: cleanup issue

### ISSUE THREAD

**Issue by lolski** [graknlabs/workbase#114]
## Description

The `assemble-windows` CI job would leave the GCP instance running under some circumstances:
- https://circleci.com/gh/graknlabs/workbase/1113
- https://circleci.com/gh/graknlabs/workbase/1124
- https://circleci.com/gh/graknlabs/workbase/1330 (cause: the workflow is cancelled, causing the `finally` block to not get executed)
- https://circleci.com/gh/graknlabs/workbase/976

The exact causes vary and the first step is for us to know the scenarios under which the job would fail to clean up the instances.

**Comment by vmax** [comment]
No longer needed after #229
